src/components/Accordion/Accordion.mdx

```diff
@@ -19,6 +21,7 @@ import Link from "../Link/Link";
 - {<Link href="#accessibility" target="_self">Accessibility</Link>}
 - {<Link href="#faq-example" target="_self">FAQ Example</Link>}
 - {<Link href="#example-with-panelmaxheight-prop" target="_self">Example with panelMaxHeight Prop</Link>}
+- {<Link href="#changelog" target="_self">Changelog</Link>}
```

Comment:
In the "Overview" section, the second paragraph mentions the array of objects that contains `label`, `panel`, `accordionType`. This should also include the new `ariaLabel` and `buttonInteractionRef` properties.

Reply:
I added details about `ariaLabel`, but I will allow @gonuguntla to add details about `buttonInteractionRef`.